connect_labs/labs/indicators/tests/test_policy.py

```
@@ -183,3 +183,74 @@ def test_the_partial_country_does_not_carry_its_unevaluated_regions_population(s
+        country, county, district = self._liberia()
+        set_value(country, "ors_coverage", 20, source=Source.DHS)
+        set_value(county, "ors_coverage", 45, source=Source.DHS)
+
+        bulk = BulkResolver([district])
+        one = resolve("ors_coverage", district)
+        many = bulk.get("ors_coverage", district)
+
+        assert (many.value, many.measured_at.pk) == (one.value, one.measured_at.pk)
+
+    def test_a_district_still_reaches_its_country_when_the_province_has_nothing(self):
+        country, county, district = self._liberia()
```

Ruff (0.16.3)

[warning] Unpacked variable `county` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)